test/e2e/suites/e2e/clusterctl_upgrade_test.go

```diff
@@ -28,11 +28,6 @@ import (
 	"sigs.k8s.io/cluster-api-provider-openstack/test/e2e/shared"
 )
 
-// Note: There isn't really anything fixing this to the v0.6 release. The test will
-// simply pick the latest release with the correct contract from the E2EConfig, as seen here
-// https://github.com/kubernetes-sigs/cluster-api/blob/3abb9089485f51d46054096c17ccb8b59f0f7331/test/e2e/clusterctl_upgrade.go#L265
-// When new minor releases are added (with the same contract) we will need to work on this
-// if we want to continue testing v0.6.
```

Comment:
This comment is no longer relevant. It is now possible to set the exact version to use for each provider and we are using this below in the test.